vitest.config.mjs

```diff
@@ -0,0 +1,23 @@
+import { defineConfig } from 'vitest/config';
+import { dirname, resolve } from 'path';
+import { fileURLToPath } from 'url';
+
+const __filename = fileURLToPath(import.meta.url);
+const __dirname = dirname(__filename);
+
+// https://vitest.dev/config/
+export default defineConfig({
+  resolve: {
+    alias: {
+      _: resolve(__dirname),
+      '@': resolve(__dirname, './src'),
+    },
```

Comment:
There's probably an opportunity to DRY this up, but it's fine to duplicate aliases for now.

Reply:
Agreed; fine for now